static/gsApp/components/navBillingStatus.spec.tsx

```diff
@@ -11,17 +11,42 @@ import ConfigStore from 'sentry/stores/configStore';
 import PrimaryNavigationQuotaExceeded from 'getsentry/components/navBillingStatus';
 import SubscriptionStore from 'getsentry/stores/subscriptionStore';
 
+// Jun 06 2022
+const MOCK_TODAY = 1654492173000;
+
+jest.mock('moment-timezone', () => {
```

Comment:
Have you tried using `setMockDate()` in `sentry-test/utils`?

Reply:
i have not LOL i didn't know that existed, let me make that change!